perfmetrics/scripts/fio/fio_metrics.py

```diff
@@ -152,75 +278,37 @@ def _load_file_dict(self, filepath) -> Dict[str, Any]:
       raise NoValuesError(f'JSON file {filepath} returned empty object')
     return fio_out
 
-  def _extract_metrics(self, fio_out) -> List[Dict[str, Any]]:
-    """Extracts and returns required metrics from fio output dict.
-
-      The extracted metrics are stored in a list. Each entry in the list is a
-      dictionary. Each dictionary stores the following fio metrics related
-      to a particualar job:
-        jobname, filesize, number of threads, IOPS, Bandwidth and latency (min,
-        max and mean)
+  def _get_start_end_times(self, out_json, job_params) -> List[Tuple[int]]:
+    """Returns start and end times of each job as a list.
 
     Args:
-      fio_out: JSON object representing the fio output
+      out_json : FIO json output
+      job_params: List of dicts, Parameters of all jobs
```

Comment:
do you mean each dictionary contains parameter of a job.  please rephrase the comment, its not clear.

Reply:
done